### ISSUE TITLE
Unable to pull songs

### ISSUE THREAD

**Issue by Shadowblight** [pvrs12/Anesidora#106]
Did some hardware updating recently, and now I'm using the rear jack for my audio instead of the front jack, but still same motherboard.

It was working prior using Browsec VPN Firefox extension to smartfilter filter from pandora.com to US.

Now however, it just keeps flipping to new songs, never playing any one song. I can go direct to the site in the browser and it works just fine.

FF version 123.0, up-to-date. Using portable version, if it matters.

**Comment by hucario** [comment]
I don't have a ton of time today, but I think I can investigate this tomorrow.
For now, are there any errors in the extension console?
You should be able to access it by going to about:debugging, then hitting "inspect" on Anesidora.

**Comment by Shadowblight** [comment]
That'd be amazing if you could help.

Yeah I've got a few errors rolling around in the console. They look like this:

![image](https://github.com/pvrs12/Anesidora/assets/75965029/f682294f-44e2-4b24-aea6-ee57700def30)

Please let me know if you need anything else. I'd be really happy to get it working again.

**Comment by hucario** [comment]
Looking at the logs, it looks like every time the browser requests an audio file, the server responds with a 499 code. After it eventually runs out of options, the HTMLAudioElement throws an error, as it has no valid sources. Anesidora handles the error by saying "*this* song doesn't work at all, maybe the next one will?", skipping / fetching another song, and giving the browser another set of urls to try, which of course all fail and cause the process to repeat.  I... have no idea what would cause this, to be honest. Apparently it's an nginx error involving the client closing the connection before the server has sent a response, but we don't manually handle audio fetching in Anesidora, the browser does that.
If you are behind a proxy, it's possible that it's turning a 403 ("unauthorized") into a 499. We check for a 403 and recognize you need to relog when a 403 is seen, but that wouldn't run on a 499. Maybe try logging out and in again?

**Comment by Shadowblight** [comment]
Really bizarre... I did a full uninstall and reinstall of both Anesidora and also Browsec extensions just to be safe. Did a full purge on cookies for pandora in the browser and restarted even, but it was still doing the same thing, after adding "pandora.com" in for the domain (this is what it was before).

I noticed my Kapersky was crying about going to "pandora.com" now directly though. So I tried www.pandora.com, same thing. I could access it through google search link, which had the https:// on it. Well... I tried pasting the full https;//www.pandora.com that google results gave me into Browsec, it truncated to just "www.pandora.com", and NOW Anesidora works. I have no clue what on Earth what spawned this series of events, because none of my software has changed besides installing a video card driver-- certainly nothing having to do with my browser/extensions!! LOL

Thanks for being champ to lead me in the right direction, regardless!
